Dockerfile-fpm

```diff
@@ -1,5 +1,7 @@
 FROM php:7.2-fpm-alpine3.7 as fpm
 
+RUN rm /usr/local/etc/php-fpm.d/*.conf
```

Comment:
What's the before and after files?
As there's the docker.conf, zz-docker.conf I'm not sure if this is a too safe of a change?

Reply:
they are copied in on live 48.

Reply:
putting this `rm` on line 47 would be better, for context.